Python/Text_Extract/text_extract.py

```diff
@@ -0,0 +1,43 @@
+from PIL import Image
+import pytesseract as pt
+import os
+from pathlib import Path
+
+
+current_location = (os.getcwd() + '\\')
+
+
+def main():
+    # Enter the name of folder which contains img files
+    image_location = input("Enter the Folder name containing Images: ")
+    image_path = os.path.join(current_location, image_location)
+    # Enter the name of folder which would contain respective txt files
+    destination = input("Enter your desired output location: ")
+    destination_path = os.path.join(current_location, destination)
+    # Path to Tesseract
+    pt.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract'
```

Comment:
Can this be changed to make sure that the Users don't run into any issues especially cross-platform?

Reply:
Shheessss, why is that stupid line here XD
My bad

Reply:
So here is the thing, as far as I know, there are 3 ways to sort this

- Either take the path as input from the user (Too much user input, but simple for beginners :D)
- Put tesseract file in the directory itself (I don't like this at all)
- PATH env variable pointing to Tessaract (Best method, but can be daunting for some users)

I suggest taking user input from the person, but In `README` i can link a method to update the PATH variable.
-

Reply:
Yeah. Please do the same 👍 

I expect we need to dedicate sometime to help the existing Scripts a bit less daunting to use and access.

Reply:
Will do the same

Reply:
I understand your concern regarding code quality, although I don't think its the perfect idea, I think my latest comments on #584 might give us a start

Reply:
Yeah. We have already started working on the same.